src/containers/Comments/Comments.js

```diff
@@ -1,34 +1,73 @@
-import React from 'react'
+import React, {  useRef, useState } from 'react'
 import { useTranslation } from 'react-i18next'
-import { View } from 'react-native'
+import { ScrollView, View } from 'react-native'
+import * as Device from 'expo-device'
 
 import { TextSourceSans } from '../../components/TextSourceSans'
 
 import { Comment } from './Comment'
 
+const getScrollOffset = () => {
+  if (Device.osName === 'Android') {
+    return 180
+  } else if (Device.osName === 'iOS') {
+    return 210
+  } else if (Device.osName === 'iPadOS') {
+    return 230
+  }
+}
+
 export const Comments = (props) => {
+  const commentsListRef = useRef()
+  const [commentViewInfo, setCommentViewInfo] = useState([])
   const { t } = useTranslation()
   const { navigation } = props
 
+  // store position of <Comment> and comment id, for later
+  // this happens on mount of this component
+  const handleOnLayout = (index, commentId, y, height) => {
+    commentViewInfo[index] = {commentId, y, height}
+    setCommentViewInfo(commentViewInfo)
+  }
+
+  // after click on 'reply', this "inbetween" handler will be called to
+  // find the comment which has been clicked on. Then the parent's
+  // handleReply will be called.
+  const handleReplyLayout = (commentContentType, commentObjectPk) => {
+    const parentComment = commentViewInfo
+      .find(cvc => cvc.commentId === commentObjectPk)
+    props.handleReply(
+      commentContentType,
+      commentObjectPk,
+      parentComment.y + parentComment.height + getScrollOffset()
```

Comment:
@philli-m i checked the height of the comment form, as you suspected it could be that, but the actual hight of the form is 70 (iOS) and 80 (android). So it would be too less and cutoff weirdly.

With a fixed offset (different for each OS) it works for the comment being expanded (read more / read less).

Reply:
something goes a little off here as there is now no way to see the other replies unless you add one yourself